### ISSUE TITLE
Jedi: Like button `aria-pressed` dynamic signal

### ISSUE THREAD

**Issue by gihrig** [gihrig/SolidStart-Demo#8]
- Action dialog not working
- There is no action placeholder for this toggle button
- It should have a toggle feature suitable for back-end update
- Should have appropriate color or ring indicator to show selected
- Should have corresponding `aria-pressed` status
